### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

**Comment by gallegonovato** [comment]
Good afternoon @nickspaargaren

I forgot to tell you why I made those changes to the readme links.

I made them because Jorgu's gitlab no longer exists. And those are the new links on the platform I migrate things to.

Best regards

File: README.md [hunk 1]
```diff
@@ -127,12 +127,12 @@ python3 convert.py json | jq -r '(.Analytics,.Doubleclick) | .[]'
 ## Can I block the other letters of GAFAM ?
 Of course, here’s some filterlists that should help you accomplish that.
 
-**A**pple : https://github.com/c-edw/ios-telemetry or https://github.com/1r2/iosparanoid
+**A**pple : https://github.com/c-edw/ios-telemetry , https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
 
-**F**acebook : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/facebook/all or https://raw.githubusercontent.com/anudeepND/blacklist/master/facebook.txt
+**F**acebook : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/facebook/all , https://raw.githubusercontent.com/anudeepND/blacklist/master/facebook.txt or https://www.github.developerdan.com/hosts/lists/facebook-extended.txt
```
Comment (nickspaargaren) [inline README.md:132]:
```suggestion
**F**acebook : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/facebook/all, https://raw.githubusercontent.com/anudeepND/blacklist/master/facebook.txt or https://raw.githubusercontent.com/lightswitch05/hosts/master/docs/lists/facebook-extended.txt
```

File: README.md [hunk 2]
```diff
@@ -127,12 +127,12 @@ python3 convert.py json | jq -r '(.Analytics,.Doubleclick) | .[]'
 ## Can I block the other letters of GAFAM ?
 Of course, here’s some filterlists that should help you accomplish that.
 
-**A**pple : https://github.com/c-edw/ios-telemetry or https://github.com/1r2/iosparanoid
+**A**pple : https://github.com/c-edw/ios-telemetry , https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
```
Comment (nickspaargaren) [inline README.md:130]:
```suggestion
**A**pple : https://github.com/c-edw/ios-telemetry, https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
```

File: README.md [hunk 3]
```diff
@@ -127,12 +127,12 @@ python3 convert.py json | jq -r '(.Analytics,.Doubleclick) | .[]'
...
 
-**F**acebook : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/facebook/all or https://raw.githubusercontent.com/anudeepND/blacklist/master/facebook.txt
+**F**acebook : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/facebook/all , https://raw.githubusercontent.com/anudeepND/blacklist/master/facebook.txt or https://www.github.developerdan.com/hosts/lists/facebook-extended.txt
 
 **A**mazon : https://github.com/nickspaargaren/no-amazon
 
-**M**icrosoft : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/microsoft/all or https://gitlab.com/Jorgu81/hosts/raw/master/Microsoft/Microsoft and https://gitlab.com/Jorgu81/hosts/raw/master/Microsoft/SpyMicrosoft
+**M**icrosoft : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/microsoft/all , https://git.nixnet.services/Narsil/hosts/raw/branch/master/Microsoft/Microsoft and https://git.nixnet.services/Narsil/hosts/raw/branch/master/Microsoft/SpyMicrosoft or https://codeberg.org/Cyanic76/Hosts/src/branch/pages/corporations/Microsoft
```
Comment (nickspaargaren) [inline README.md:136]:
```suggestion
**M**icrosoft : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/microsoft/all
```

**Review by nickspaargaren — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gallegonovato) [reply README.md:140]: Change made, if something is wrong do not hesitate to let me know.

File: README.md (hunk 2 above)
Comment (gallegonovato) [reply README.md]: Change made, if something is wrong do not hesitate to let me know.

File: README.md [hunk 4]
```diff
@@ -127,12 +127,31 @@ python3 convert.py json | jq -r '(.Analytics,.Doubleclick) | .[]'
...
+Of course, here’s some filterlists that should help you accomplish that.
+
+**A**pple : 
+https://github.com/c-edw/ios-telemetry , https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
+
+**F**acebook : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/facebook/all, https://raw.githubusercontent.com/anudeepND/blacklist/master/facebook.txt or https://raw.githubusercontent.com/lightswitch05/hosts/master/docs/lists/facebook-extended.txt
+
+**A**mazon : https://github.com/nickspaargaren/no-amazon
+
+**M**icrosoft : /https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/microsoft/all
```
Comment (nickspaargaren) [inline README.md:155]:
```suggestion
**M**icrosoft : https://raw.githubusercontent.com/jmdugan/blocklists/master/corporations/microsoft/all
```

File: README.md [hunk 5]
```diff
@@ -127,12 +127,13 @@ python3 convert.py json | jq -r '(.Analytics,.Doubleclick) | .[]'
 ## Can I block the other letters of GAFAM ?
 Of course, here’s some filterlists that should help you accomplish that.
 
-**A**pple : https://github.com/c-edw/ios-telemetry or https://github.com/1r2/iosparanoid
+**A**pple : 
+https://github.com/c-edw/ios-telemetry, https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
```
Comment (nickspaargaren) [inline README.md:131]:
```suggestion
**A**pple : https://github.com/c-edw/ios-telemetry, https://github.com/1r2/iosparanoid or https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/native.apple.txt
```

**Comment by nickspaargaren** [comment]
Thanks @gallegonovato, looks great!

**Comment by gallegonovato** [comment]
Good afternoon @Nickspaargaren

I'm glad you didn't screw up. Since I don't know why when I posted a link I got the notification to your reply, or something else.

But in the end I was able to get my hands on it. It's just that these changes they are making in github, personally I don't like them. 

Since for example to put a connection takes longer than before. Because before it allowed you to search while editing. But now, at least for me it doesn't allow it.

Best regards

**Comment by gallegonovato** [comment]
Good afternoon @nickspaargaren

One thing I just realized. The links that I had put for Microsoft, are more current than those of jmdugan.

Since the ones I posted for Narsil. This one hates all weird connections and hates Microsoft.

And jmdugan's have not been updated for 3 years, you can see it at: https://github.com/jmdugan/blocklists/tree/master/corporations

So there will be new connections that will not appear on that host.

I advise to change it for a more current one.

A hug

**Pull request merged**